python/pyarrow/tests/test_table.py

```diff
@@ -1577,10 +1577,13 @@ def test_table_repr_to_string():
                        metadata={b'foo': b'bar'})
 
     tab = pa.table([pa.array([1, 2, 3, 4], type='int16'),
-                    pa.array([1, 2, 3, 4], type='int32')], schema=schema)
+                    pa.array([10, 20, 30, 40], type='int32')], schema=schema)
     assert str(tab) == """pyarrow.Table
 c0: int16
-c1: int32"""
+c1: int32
+----
+c0: [[1,2,3,4]]
+c1: [[10,20,30,40]]"""
```

Comment:
If the table has many rows, will all their values be printed or will there be an ellipsis?

Reply:
Cool, can you add a test for that?

Reply:
I'll add a test, originally tought it wasn't necessary as it just invokes `to_string` of the aray and we already have tests for that.